skbio/stats/composition.py

```diff
@@ -642,6 +642,128 @@ def ilr_inv(mat, basis=None, check=True):
     return clr_inv(np.dot(mat, basis))
 
 
+@experimental(as_of="0.5.1-dev")
+def alr(mat, denominator_idx=0):
+    r"""
+    Performs additive log ratio transformation.
+
+    This function transforms compositions from a D-part Aitchison simplex to
+    a non-isometric real space of D-1 dimensions. The argument
+    `denominator_col` defines the index of the column used as the common
+    denominator. The :math: `alr` transformed data are amenable to multivariate
+    analysis as long as statistics don't involve distances.
+
+    :math:`alr: S^D \rightarrow \mathbb{R}^{D-1}`
+
+    The alr transformation is defined as follows
+
+    .. math::
+        alr(x) = [log \frac{x_1}{x_D}, \ldots, log \frac{x_{D-1}}{x_D}]
```

Comment:
By convention, we've been using the natural log to denote logs.  The above statement can be rephrased as
```
alr(x) = [\ln \frac{x_1}{x_D}, \ldots, \ln \frac{x_{D-1}}{x_D}]
```
It'll also render better :)